string command_type
float64[] X_initial
float64[] X_goal

